app/src/main/res/values/strings.xml

```diff
@@ -65,10 +65,14 @@
 
     <!-- Private mode shortcut "contextual feature recommendation" (CFR) -->
     <!-- Text for the main message -->
-    <string name="cfr_message">Add a shortcut to open private tabs from your Home screen.</string>
+    <string name="cfr_message" moz:removedIn="109" tools:ignore="UnusedResources">Add a shortcut to open private tabs from your Home screen.</string>
+    <!-- Text for the main CFR message describing that the next private tab can be launched in one tap -->
```

Comment:
```suggestion
    <!-- Text for the Private mode shortcut CFR message for adding a private mode shortcut to open private tabs from the Home screen -->
```
I am trying to avoid describing the string by repeating the actual string. Open any changes you might see from this feedback

Reply:
Updated! I wasn't sure whether the description or the string itself were used more closely during translation, and I didn't want to end up in a scenario where the string was translated to match the update. This makes sense to me though

Reply:
I saw you were using `Private Browsing` whereas I used `Private mode`. Feel free to update this as well for consistency.